geode-core/src/distributedTest/java/org/apache/geode/cache/query/dunit/PersistentRegionCompactRangeIndexDUnitTest.java

```diff
@@ -151,6 +151,50 @@ public void giiWithPersistenceAndStaleDataDueToUpdatesShouldCorrectlyPopulateInd
     });
   }
 
+  @Test
+  public void giiWhereNewAndOldValueIsTombstoneShouldNotThrowNPE()
+      throws Exception {
+    // This test requires the oldKeyValuePair to be instantiated, to do so, we need the gii thread
+    // to gii some updates or removes on existing (non-tombstone) values before processing the
+    // scenario of tombstone to tombstone
+    String regionName = "replicateTestRegion";
```

Comment:
Looks fine to me. I was just wondering about the naming in our tests. This region name has "replicated" in it but the other tests have regions names like "persistentTestRegionWithEntrySetIndex" which does not tell us what region type it is unless we view the properties file.